examples/js/postprocessing/EffectComposer.js

```diff
@@ -60,6 +60,22 @@ THREE.EffectComposer = function ( renderer, renderTarget ) {
 
 	this.clock = new THREE.Clock();
 
+	// for VR
+
+	var rendererSize = new THREE.Vector2();
+	var scope = this;
+
+	function onSessionStateChange() {
+
+		renderer.getDrawingBufferSize( rendererSize );
+		scope.setPixelRatio( renderer.getPixelRatio() );
+		scope.setSize( rendererSize.x, rendererSize.y );
```

Comment:
I appears you are scaling by pixel ratio twice here.

Reply:
The addition of `setPixelRatio` after `getDrawingBufferSize` was made in the previous PR (commit f0fdd62) and I didn't change it. But it seems you're right. I'll change `getDrawingBufferSize` to `getBufferSize`.